### ISSUE TITLE
Introduce fee thresholds for altcoins

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-firmware#1191]
Follow up of #1087 where we have realized we do not have fee threshold checks for altcoins. Let's add both soft and hard limit for all supported coins. Product suggests 10x for an average fee cost for the soft limit and 100x for the hard limit. Let's discuss this in Triage though.

**Comment by prusnak** [comment]
This has been fixed via https://github.com/trezor/trezor-firmware/commit/f51fac34109db730d5de0fc66af4fdc83ca09c32 right?

**Comment by tsusanka** [comment]
No. In here I meant altcoins that are not Bitcoin-based (ETH, NEM, XRP etc.). Let me rephrase the issue.
